easy/13.js

```diff
@@ -1,9 +1,9 @@
 function isPrefix(word, prefix) {
-	
+    return word.includes(prefix.substring(0, prefix.length - 1));

```

Comment:
This is not working for isPrefix('arachnophobia', 'no-')
expected false, Actual true